telemetry/sql/000009_peer_count.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS peerCount (
```

Comment:
would we have table per metric?

Reply:
That is what we have now - each metric type gets own table - receivedEnvelopes, pushedEnvelopes...

https://github.com/status-im/telemetry/blob/master/telemetry/sql/000005_pushed_envelope.up.sql